socorro/unittest/cron/setup_configman.py

```diff
@@ -79,9 +79,7 @@ def get_config_manager_for_crontabber(
     else:
         more_definitions = [CronTabberApp.get_required_config()]
 
-    local_overrides = {
-        'logger': Mock(),
-    }
```

Comment:
This line has been in the codebase since this file was written. However, I think it's wrong. The configuration doesn't have a `logger` object--it has a bunch of `logging-blahblahblah` things. Other test setup functions toss in a logger object after the configuration is configurized.

Fixing this makes the warning go away because then the expected keys matches the given keys.